src/ZkFold/Symbolic/Data/FieldElement.hs

```diff
@@ -102,30 +97,3 @@ instance Symbolic c => BinaryExpansion (FieldElement c) where
   fromBinary bits =
     FieldElement $ symbolicF bits (Par1 . foldr (\x y -> x + y + y) zero)
       $ fmap Par1 . horner . fromVector
-
-instance Symbolic c => DiscreteField' (FieldElement c) where
```

Comment:
as `Discrete` and `Trichotomy` stuff is not needed, maybe we can then remove them from our typeclass hierarchy

Reply:
Discussed in #154